biomage/experiment/utils.py

```diff
@@ -141,84 +141,19 @@ def get_cognito_username(email):
 
 def get_experiment_project_id(experiment_id, source_table):
 
-    table = boto3.resource('dynamodb').Table(source_table)
+    table = boto3.resource("dynamodb").Table(source_table)
 
     project_id = table.get_item(
-        Key={"experimentId": experiment_id},
-        ProjectionExpression='projectId'
-    ).get("Item")['projectId']
+        Key={"experimentId": experiment_id}, ProjectionExpression="projectId"
```

Comment:
I preferred the old formatting with each argument on a newline

Reply:
auto-formatter sets it like that, can't change